packages/gatsby/.size-limit.js

```diff
@@ -0,0 +1,7 @@
+module.exports = [
+  {
+    path: `cache-dir/*.js`,
+    webpack: false,
```

Comment:
So I _almost_ solved this, but ultimately kept running into babel issues. The approach I tried was:

- add a `.size-limit.config.js` webpack config file
- the webpack config file adds the babel loader with our preset
- add a `config: './.size-limit.config'` property to `.size-limit.js`
- failure...

Fails with

```
 ERROR  Size Limit can't resolve
        @babel/runtime/helpers/defineProperty
        in /Users/dschau/Code/Work/gatsby/gatsby/packages/gatsby/cache-dir
```

Reply:
Here's `.size-limit.config.js` for posterity

```js
const path = require(`path`)
const { module: browserModule } = require(`./package.json`)
const preset = require(`./babel.config`)

module.exports = {
  entry: path.join(__dirname, browserModule),
  module: {
    rules: [
      {
        test: /\.js$/,
        use: [
          {
            loader: require.resolve(`babel-loader`),
            options: preset
          }
        ]
      }
    ]
  }
}
```